doc/news/changes/minor/20191012PeterMunchDanielArndt

```diff
@@ -0,0 +1,4 @@
+New: Introduce a base class for the VectorizedArray specializations, containing 
+common functionalities (e.g., size, iterator).
```

Comment:
Maybe one suggestion to keep the changelog a bit compact: Can you merge the two changelogs you have and merge it with mine from this week here https://github.com/dealii/dealii/blob/master/doc/news/changes/minor/20191008MartinKronbichler and say that we added some STL-like functionality like an iterator and size?

Reply:
Done!